tests/transfer-item.test.js

```diff
@@ -0,0 +1,120 @@
+const { connect, JSONCodec } = require("nats");
+const { JEST_TIMEOUT = 20000 } = process.env;
+
+describe("Given Tokenization Service is connected to NATS", () => {
+    let natsConnection;
+    let jsonCodec = JSONCodec();
+
+    beforeAll(async () => {
+        natsConnection = await connect();
+    });
+
+    describe("And there is an available item", () => {
+        let response;
+        let newItemId = 27;
+
+        beforeAll(async () => {
+            jest.setTimeout(JEST_TIMEOUT);
+            response = await natsConnection.request(
+                "tokenization-service.add_warehouse_item",
+                jsonCodec.encode({
+                    data: {
+                        XP: "97"
+                    },
+                    item_id: newItemId,
+                    name: "Karim Benzema",
+                    total_quantity: 9,
+                    available_quantity: 9
+                }),
+                { max: 1, timeout: JEST_TIMEOUT }
+            );
+        });
+
+        describe("When I create two inventory contracts", () => {
+            let oldInventoryAddress;
+            let newInventoryAddress;
+
+            beforeAll(async () => {
+                jest.setTimeout(JEST_TIMEOUT);
+                oldInventoryResponse = await natsConnection.request(
+                    "tokenization-service.create_inventory",
+                    undefined,
+                    { timeout: JEST_TIMEOUT }
+                );
+
+                newInventoryResponse = await natsConnection.request(
+                    "tokenization-service.create_inventory",
+                    undefined,
+                    { timeout: JEST_TIMEOUT }
+                );
+
+                oldInventoryAddress = jsonCodec.decode(
+                    oldInventoryResponse.data
+                ).inventory_address;
+
+                newInventoryAddress = jsonCodec.decode(
+                    newInventoryResponse.data
+                ).inventory_address;
+            });
+
+            describe("When I assign this item to the old inventory", () => {
+                beforeAll(async () => {
+                    jest.setTimeout(JEST_TIMEOUT);
+                    response = await natsConnection.request(
+                        "tokenization-service.assign_inventory_item",
+                        jsonCodec.encode({
+                            inventory_address: oldInventoryAddress,
+                            item_id: newItemId,
+                            instance_number: 1
+                        }),
+                        { timeout: JEST_TIMEOUT }
+                    );
+                });
+                
+                it("Then reflects the request back", () => {
+                    expect(jsonCodec.decode(response.data)).toEqual({
```

Comment:
same here, we've already tested this, so let's remove the `it`.
You will have `describe` without any `it` which is going to return you an error, that means you need to remove the describe on line 82. Make sure the Given/When/Then still make sense when reading them after removing the describe